tests/sentry/buffer/test_redis.py

```diff
@@ -248,6 +249,63 @@ def test_process_pending_partitions_none(self, process_pending, process_incr):
         # Make sure we didn't queue up more
         assert len(process_pending.apply_async.mock_calls) == 2
 
+    def test_enqueue(self):
+        PROJECT_ID_BUFFER_LIST_KEY = "project_id_buffer_list"
+        project_id = 1
```

Comment:
I did this because it seems weird to actually create rules and events in a redis buffer test file, and the functions I added aren't specific to those necessarily (though that will be their only use for now).  I could be convinced to go about this differently, but I wanted to test the input and output.

Reply:
Agree, I actually kinda like that this test isn't explicitly using those objects tbh. This makes it a little clearer that you can build a data structure to store in redis.

Once we start using this buffer, maybe on those tests we actually create the project etc to use.